test/realtime/message.test.js

```diff
@@ -1183,5 +1183,119 @@ define(['ably', 'shared_helper', 'async', 'chai'], function (Ably, helper, async
           });
       });
     }
+
+    it('subscribes to filtered channel', function (done) {
+      var testData = [
+        {
+          name: 'filtered',
+          data: 'This message can be filtered',
```

Comment:
For clarity I think this would be better
```suggestion
          data: 'These headers match the JMESPath expression so this message should not be filtered',
```